### ISSUE TITLE
Training duration on A100

### ISSUE THREAD

**Issue by kirrukirru** [XPixelGroup/DiffBIR#46]
I started training with about 2000 images with a batch size of 10.
 1. How long does traning take for 2000 images set? Currently, it is at about 800 Epoch and each Epoch taking about 2.5 mins. Doesn't show total Epochs to process.
 2. I can see files like `step=49999.ckpt` etc. are created after every 10000 steps. If the training is stopped and started again, will it resume from where it was stopped?
 3. Can the training be done on CPU only?

Thanks,
Kiran

**Comment by 0x3f3f3f3fun** [comment]
Thanks for your questions.
1. A dataset of 2000 images is relatively small, and I believe the training should not take too long, as it could lead to overfitting. I cannot provide you with a specific training steps, you can determine it based on the performance on the test set.
2. No, you should set the parameter `resume` to the path of a `step` file you want to resume.
3. I haven't used CPU for training, and it is not recommended.

**Comment by kirrukirru** [comment]
Thanks for the reply!
I have set the resume path the last `step` file path in the config file and strated the training; It started from Epoch0 and new checkpoints version added in `lightning_logs` folder. Is it expected?

**Comment by 0x3f3f3f3fun** [comment]
Yes, it's expected.
